test/test_uri.jl

```diff
@@ -1,4 +1,4 @@
-using URIParser
```

Comment:
Could probably test both? Though I guess we'd need to use the namespaces appropriately.